cmd/preview/main.go

```diff
@@ -29,6 +30,5 @@ func main() {
 			}
 		}
 		log.Fatal(err.Error())
-		os.Exit(1)
```

Comment:
this is removable because `log.Fatal` exits for us, right?

Reply:
Yes exactly